drivers/interrupt_controller/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ endif()
 if(CONFIG_PLIC_SHELL)
   message(WARNING "
       WARNING:  `CONFIG_PLIC_SHELL` is enabled.
-      This can use quite a bit of RAM (PLICs * IRQs * sizeof(uint16_t))"
+      This can use quite a bit of RAM (PLICs * IRQs * sizeof(PLIC_IRQ_COUNT_TYPE))"
```

Comment:
Unless there is a significant reason to change from `uint16_t` within this PR, I would say it's simpler to keep the original type.

Reply:
When SMP IPI is done over PLIC, it overflows the `uint16_t` very quickly, so I thought that it might be nice to make it configurable, but I have removed that from this PR to reduce noice